etc/ncp-config.d/DDNS_spDYN.sh

```diff
@@ -122,11 +128,11 @@ configure()
     
     # Adds file to cron to run script for DNS record updates and change permissions 
     touch $CRONFILE
-    echo "0 * * * * root $INSTALLPATH/spdnsUpdater.sh $DOMAIN_ $TOKEN_ >/dev/null 2>&1" > "$CRONFILE"
+    echo "0 * * * * root $INSTALLPATH/spdnsUpdater.sh $DOMAIN_ $TOKEN_ $IPv6 >/dev/null 2>&1" > "$CRONFILE"
```

Comment:
shouldn't this be `$IPv6_`?

Reply:
Thats true.
I did the changes on my device and in Github in parallel – and sometimes forgot to update on both channels.
Thanks for correcting.